web/frontend/src/components/slam-2d-render.vue

```diff
@@ -348,10 +346,16 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
     axesNeg
   );
 
-  if (props.pose !== undefined) {
-    updatePose(props.pose!);
+  // re-add base and destination marker so they render on top of all other objects
+  const destinationMarker = scene.getObjectByName('DestinationMarker');
+  if (destinationMarker !== undefined) {
+    scene.add(destinationMarker);
```

Comment:
1. Why do we need to re add them?
2. Where are they being removed from the scene?

Reply:
Let's confirm with @micheal-parks whether or not this is the right way to do this.

I would be kinda surprised if the only way to render a three.js entity behind another is to re add every entity that needs to be in front of it.

Reply:
Sry, didn't see your comment before I posted my last one @micheal-parks .